### ISSUE TITLE
[Request] Allow selecting and downloading a second stream for audio

### ISSUE THREAD

**Issue by smares** [chandler-stimson/live-stream-downloader#74]
Some sites have two separate streams for audio and video. It would be nice if the stream selection dialog would allow selecting a second stream to download for audio in parallel and have it muxed automatically into the final mkv. Currently, I have to download the streams separately and mux them with ffmpeg.

**Comment by chandler-stimson** [comment]
For sequential downloading, there is a button you can press after selecting both streams. To enable parallel downloading, click the action button again and select the other stream in the new dialog.

**Comment by PatrykRoma** [comment]
@smares # @chandler-stimson @chandler-stimson @somsak 
/ #
| Header | Header | Header | Header | Header |
|--------|--------|--------|--------|--------|
| Cell | Cell | Cell | Cell | Cell |
| Cell | Cell | Cell | Cell | Cell |
| Cell | Cell | Cell | Cell | Cell |
| Cell | Cell | Cell | Cell | Cell |
| Cell | Cell | Cell | Cell | Cell |

**Comment by PatrykRoma** [comment]
| Header |
|--------|
| Cell |
| Cell | 
Duplicate of #

> ****

**Comment by Zero3K** [comment]
It still doesn't mux the two streams after they are downloaded.
